### ISSUE TITLE
[0.3.0] flow@0.38.0 reports unused suppression

### ISSUE THREAD

**Issue by avaragado** [rpl/flow-coverage-report#24]
I've added flow-coverage-report to my project. I'm using flow@0.38.0, and it's not configured to ignore node_modules. (I believe this is best practice.)

Flow reports this:

```
node_modules/flow-coverage-report/src/lib/flow.js:87
 87:   // $FLOW_FIXME: code is there, but flow doesn't seem to know about it.
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Error suppressing comment. Unused suppression
```

I guess an earlier version of flow barfed at the next line. It isn't now, so flow is now differently barfing.

Removing line 87 stops the error, which I guess is the correct fix.

**Comment by avaragado** [comment]
BTW, my `.flowconfig` contains:

```
suppress_comment=.*\\$FLOW_FIXME
```

Without this line, flow@0.38.0 reports this:

```
node_modules/flow-coverage-report/src/lib/report-html.js:94
 94:         react.renderToStaticMarkup(new FlowCoverageHTMLReport({
                                        ^ prototype. This type is incompatible with the expected param type of
273:     element: React$Element<any>
                  ^^^^^^^^^^^^^^^^^^ React$Element. See lib: /private/tmp/flow/flowlib_1b0c2da7/react.js:273

node_modules/flow-coverage-report/src/lib/report-html.js:130
130:                 react.renderToStaticMarkup(new FlowCoverageHTMLReport({
                                                ^ prototype. This type is incompatible with the expected param type of
273:     element: React$Element<any>
                  ^^^^^^^^^^^^^^^^^^ React$Element. See lib: /private/tmp/flow/flowlib_1b0c2da7/react.js:273
```

If you changed your code to use the default `$FlowFixMe` (see `suppress_comment` in https://flowtype.org/docs/advanced-configuration.html) then you could remove both your `suppress_comment` lines from your `.flowconfig`, and I could remove my line too.

**Comment by rpl** [comment]
@avaragado Thanks for reporting this issue!
I opted to update flow-bin and fix the issue related to the suppress comments (#25), which should fix the issue that you describe here.

I merged this #25 on master (it is not yet released on npm, but in the meantime you can use the fixed version by switch the devDependencies to use the github repo).

**Comment by avaragado** [comment]
Thanks!
